app/views/shared/_contact_panels.html.erb

```diff
@@ -113,111 +114,173 @@
 <% end %>
 
 <% if show_floating_chat? %>
-<a href="#" aria-hidden="true" id="js-chat-popup" class="chat-box__container chat-box__container--hidden t__floating-web-chat">
-    <div class="chat-box__heading">Web Chat</div>
-    <span class="icon icon--web-chat"></span>
+<a href="#" aria-hidden="true" id="js-chat-popup"
+  class="chat-box__container chat-box__container--hidden t__floating-web-chat">
+  <div class="chat-box__heading">Web Chat</div>
+  <span class="icon icon--web-chat"></span>
 </a>
 <% end %>
 
-<a href="#" aria-hidden="true" class="chat-popup__container" data-dough-component="ChatPopup">
+<div aria-hidden="true" class="mobile-webchat__container mobile-webchat--closed" data-dough-component="ChatPopup">
+  <button class="mobile-webchat__close">
+    <span>X</span>
+  </button>
+  <div class="mobile-webchat--icon">
     <span class="icon icon--web-chat"></span>
     <svg class="icon-svg--web-chat">
       <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-icon--web-chat"></use>
     </svg>
-</a>
+  </div>
+  <div class="mobile-webchat__form">
+    <h4><%= t('contact_panels.chat_popup.title') %></h4>
+    <div class="mobile-webchat__form-select--container">
+      <select class="mobile-webchat__form-select">
+        <option value="debt_borrowing"><%= t('contact_panels.chat_popup.dropdown_options.debt_borrowing') %></option>
+        <option value="homes_mortgages"><%= t('contact_panels.chat_popup.dropdown_options.homes_mortgages') %></option>
+        <option value="budgeting_saving"><%= t('contact_panels.chat_popup.dropdown_options.budgeting_saving') %>
+        </option>
+        <option value="work_benefits"><%= t('contact_panels.chat_popup.dropdown_options.work_benefits') %></option>
+        <option value="pensions_retirement"><%= t('contact_panels.chat_popup.dropdown_options.pensions_retirement') %>
+        </option>
+        <option value="family_care"><%= t('contact_panels.chat_popup.dropdown_options.family_care') %></option>
+        <option value="cars_travel"><%= t('contact_panels.chat_popup.dropdown_options.cars_travel') %></option>
+        <option value="insurance"><%= t('contact_panels.chat_popup.dropdown_options.insurance') %></option>
+      </select>
+    </div>
+    <div class="mobile-webchat__form-button--container">
```

Comment:
Similarly to above this should probably be marked up as ```mobile-webchat__form__button-container``` or ```mobile-webchat__button-container```. WDYT?

Reply:
@davidtrussler  I agree, I've now changed some class names in the buttons and select element so they are simpler and shorter than the previous names